### PR TITLE
Add works section

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
in this part:
I use Grid display
Flex Display

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,40 +7,191 @@
...
+        <p>A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+        <div class="tonic-lang">
+            <ul class="canopy">
+                <li><button class="tonic-button">HTML</button></li>
+                <li><button class="tonic-button">CSS</button></li>
+                <li><button class="tonic-button">JavaScript</button></li>
+            </ul>
+        </div>
+        <button class="tonic-button-submit" type="submit">See Projects</button>
+    </section>
```
Comment (tahiry-dev) [inline index.html:135]: -I suggest you to create a main container that wraps all of your projects section and where you can add display: grid. And Controller individual project alignment with flexbox

File: index.html [hunk 2]
```diff
@@ -7,40 +7,191 @@
...
+
+    <section class="about-me">
+        <hgroup class="font-sizing">
+            <h1>About <br> Myself</h1>
+        </hgroup>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+            work and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p><a href="#">LET'S CONNECT</a></p>
```
Comment (tahiry-dev) [inline index.html:145]: 'Let's connect' is another important sentence on your about me page so please use  header tag

File: index.html [hunk 3]
```diff
@@ -7,40 +7,191 @@
...
+                    </h4>
+                    <ul class="list-imag">
+                        <li>
+                            <span class="js"><img src="./images/js.svg" alt="javaScript">javaScript</span>
+                        </li>
+                        <li>
+                            <span class="css"><img src="./images/css.svg" alt="css">css</span>
+                        </li>
+                        <li>
+                            <span class="html"><img src="./images/html.svg" alt="html">html</span>
```
Comment (tahiry-dev) [inline index.html:178]: You implemented nice images here, but it is better if you use it as a background image, please

File: index.html [hunk 4]
```diff
@@ -7,40 +7,191 @@
...
+                <li><a href="https://medium.com/@tahnyybelguith"><img class="image3" src="./images/medium.svg"
+                            alt="medium"></a>
+                </li>
+                <li><a href="https://github.com/TahenyBELGUITH"><img class="image5" src="./images/github.svg"
+                            alt="github"></a>
+                </li>
+                <li><a href="#"><img class="image6" src="./images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+        </div>
+        <button class="tonic-button-submit about-btn" type="submit">Get my resume </button>
```
Comment (tahiry-dev) [inline index.html:161]: Please style your call to action button for Active, Hover, and Pressed. Please refer to the design for that

**Review by tahiry-dev — CHANGES_REQUESTED** [review]
Hello @TahenyBELGUITH 👋
You did a great job creating this mobile version of your portfolio,  but you need to update some markup and page design before moving to the next step!

## Required Changes ♻️
Please follow the design correctly, mostly in terms of alignements.
Here are examples of your pages:
seeProject
getResume
As you can see, the call to action is not aligned properly.
For a quick reminder, please look at the design
correct
- Semantic HTML are powerful tools to organize your code clearly, _Kindly check the comments under the review._

## Optional suggestions
_Although the following suggestions are optional, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (TahenyBELGUITH) [reply index.html:135]: done

File: index.html (hunk 2 above)
Comment (TahenyBELGUITH) [reply index.html:145]: done

File: index.html (hunk 3 above)
Comment (TahenyBELGUITH) [reply index.html:178]: done

File: index.html (hunk 4 above)
Comment (TahenyBELGUITH) [reply index.html:161]: done

**Review by Emmyn5600 — APPROVED** [review]
## Status: Approved :heavy_check_mark: :muscle::clap:

Hi @TahenyBELGUITH 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it :+1:.
Congratulations! :tada:

### Optional suggestions
* The button (e.g., "See project") is a link a or button tag, great work you did it however,  follow the button styling: Active, Hover and Pressed (see design).
* Try to reduce the image width for the work section to meet the mobile version from Figma.


Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**